autopush/db.py

```diff
@@ -92,6 +94,16 @@
 TRACK_DB_CALLS = False
 DB_CALLS = []
 
+# See https://botocore.readthedocs.io/en/stable/reference/config.html for
+# additional config options
+g_dynamodb = boto3.resource(
```

Comment:
What does the g_ mean?

Reply:
Well, i wanted to make sure that there was no potential for symbol conflict, and I picked "g_" because it's effectively a global.